apps/io/feeding_service_webhook.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2016 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from superdesk.io.registry import registered_webhooks
+from flask import request, abort
+import os
+import superdesk
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class FeedinServviceWebhookAuth(object):
+
+    def authenticate(self):
+        abort(403, description='You are not authorized to access this resource')
+
+    def authorized(self, allowed_roles, resource, method):
+        env_name = '{}_auth'.format(resource)
+        try:
+            auth_key = os.environ[env_name]
+        except KeyError:
+            log.error('{} environment variable is not set'.format(env_name))
+            return False
+        return request.args.get('auth') == auth_key
+
+
+class FeedingServiceWebhookResource(Resource):
+    resource_methods = ['GET', 'POST']
```

Comment:
I've put it just in case, but we can still add it in the future if needed, I'll remove